api/src/main/java/org/openmrs/notification/impl/MessageServiceImpl.java

```diff
@@ -274,7 +274,7 @@ public Message prepareMessage(String templateName, Map data) throws MessageExcep
 	 */
 	@Override
 	@Transactional(readOnly = true)
-	public List getAllTemplates() throws MessageException {
+	public List<Template> getAllTemplates() throws MessageException {
```

Comment:
In your free time, take a look here: https://github.com/openmrs/openmrs-core/pull/3368#pullrequestreview-451799315 ;)

Reply:
Got it, I've just reverted changes for MessageService and MessageServiceImpl